From: Vasudev Kamath <kamathvasudev@gmail.com>
Date: Mon, 10 Jun 2013 22:28:13 +0530
Subject: Correct the font names for nonlatin languages

Bug-Debian: http://bugs.debian.org/661245
Origin: http://lists.freedesktop.org/archives/fontconfig/2011-December/003725.html

This patch does the following.
  1. Use Rachana as serif font for Malayalam as Meera is only Sans
     family.
  2. Remove TSCu_Paranar since its legacy encoded non-unicode font.
  3. Remove non existing fonts malayalam and sampige.
---
 conf.d/65-nonlatin.conf | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/conf.d/65-nonlatin.conf b/conf.d/65-nonlatin.conf
index 3e5d1c7..71049f1 100644
--- a/conf.d/65-nonlatin.conf
+++ b/conf.d/65-nonlatin.conf
@@ -20,8 +20,6 @@
 			<family>Urdu Nastaliq Unicode</family> <!-- arabic (ur) -->
 			<family>Raghindi</family> <!-- devanagari -->
 			<family>Mukti Narrow</family> <!-- bengali -->
-			<family>malayalam</family> <!-- malayalam -->
-			<family>Sampige</family> <!-- kannada -->
 			<family>padmaa</family> <!-- gujarati -->
 			<family>Hapax Berbère</family> <!-- tifinagh -->
 			<family>MS Mincho</family> <!-- han (ja) -->
@@ -96,8 +94,6 @@
 			<family>Urdu Nastaliq Unicode</family> <!-- arabic (ur) -->
 			<family>Raghindi</family> <!-- devanagari -->
 			<family>Mukti Narrow</family> <!-- bengali -->
-			<family>malayalam</family> <!-- malayalam -->
-			<family>Sampige</family> <!-- kannada -->
 			<family>padmaa</family> <!-- gujarati -->
 			<family>Hapax Berbère</family> <!-- tifinagh -->
 			<family>MS Gothic</family> <!-- han (ja) -->
@@ -127,7 +123,6 @@
  			<family>AR PL Mingti2L Big5</family>
 			<family>ＭＳ ゴシック</family>
 			<family>ZYSong18030</family> <!-- han (zh-cn,zh-tw) -->
-			<family>TSCu_Paranar</family> <!-- tamil -->
 			<family>NanumGothic</family> <!-- hangul (ko) -->
 			<family>UnDotum</family> <!-- hangul (ko) -->
 			<family>Baekmuk Dotum</family> <!-- hangul (ko) -->
